internal/evaluator/conftest_evaluator.go

```diff
@@ -33,207 +33,201 @@ import (
-			Collections  *[]string `json:"collections,omitempty"`
-		}
-		pc := &policyConfig{}
+	type policyConfig struct {
+		NonBlocking  *[]string `json:"non_blocking_checks,omitempty"`
+		ExcludeRules *[]string `json:"exclude_rules,omitempty"`
+		IncludeRules *[]string `json:"include_rules,omitempty"`
+		Collections  *[]string `json:"collections,omitempty"`
+	}
+	pc := &policyConfig{}
 
-		// TODO: Once the NonBlocking field has been removed, update to dump the spec.Config into an updated policyConfig struct
-		if spec.Exceptions != nil {
-			log.Debug("Non-blocking exceptions found. These will be written to file", dataDir)
-			pc.NonBlocking = &spec.Exceptions.NonBlocking
+	// TODO: Once the NonBlocking field has been removed, update to dump the spec.Config into an updated policyConfig struct
+	if spec.Exceptions != nil {
+		log.Debug("Non-blocking exceptions found. These will be written to file", dataDir)
+		pc.NonBlocking = &spec.Exceptions.NonBlocking
+	}
+	if spec.Configuration != nil {
+		log.Debug("Include rules found. These will be written to file", dataDir)
+		if spec.Configuration.IncludeRules != nil {
+			pc.IncludeRules = &spec.Configuration.IncludeRules
 		}
-		if spec.Configuration != nil {
-			log.Debug("Include rules found. These will be written to file", dataDir)
-			if spec.Configuration.IncludeRules != nil {
-				pc.IncludeRules = &spec.Configuration.IncludeRules
-			}
-			log.Debug("Exclude rules found. These will be written to file", dataDir)
-			if spec.Configuration.ExcludeRules != nil {
-				pc.ExcludeRules = &spec.Configuration.ExcludeRules
-			}
-			log.Debug("Collections found. These will be written to file", dataDir)
-			if spec.Configuration.Collections != nil {
-				pc.Collections = &spec.Configuration.Collections
-			}
+		log.Debug("Exclude rules found. These will be written to file", dataDir)
+		if spec.Configuration.ExcludeRules != nil {
+			pc.ExcludeRules = &spec.Configuration.ExcludeRules
 		}
-		// Check to see that we've actually added any values to the policyConfig struct.
-		// If so, we'll update the config map. Otherwise, this is skipped.
-		if (policyConfig{} != *pc) {
-			config["config"] = map[string]interface{}{
-				"policy": pc,
-			}
+		log.Debug("Collections found. These will be written to file", dataDir)
+		if spec.Configuration.Collections != nil {
+			pc.Collections = &spec.Configuration.Collections
+		}
+	}
+	// Check to see that we've actually added any values to the policyConfig struct.
+	// If so, we'll update the config map. Otherwise, this is skipped.
+	if (policyConfig{} != *pc) {
+		config["config"] = map[string]interface{}{
+			"policy": pc,
 		}
 	}
 
-	jsonData, marshalErr := json.MarshalIndent(config, "", "    ")
-	if marshalErr != nil {
+	configJSON, err := json.MarshalIndent(config, "", "    ")
+	if err != nil {
 		return err
 	}
 	// Check to see if the data.json file exists
-	exists, err = afero.Exists(utils.AppFS, dataFilePath)
+	exists, err := afero.Exists(fs, configFilePath)
```

Comment:
More of a golang question than a review comment.. It seems like like the exists var was never declared here. How does that work?

Reply:
the `afero.Exists()` method returns two values, a boolean and an error. This is just assigning those returns to variable names to evaluate further.